.github/workflows/version.yml

```diff
@@ -44,24 +39,22 @@ jobs:
           echo "NEXT_VERSION=$NEXT_VERSION" >> $GITHUB_ENV
           echo "NEXT_VERSION_UNDERSCORE=$NEXT_VERSION_UNDERSCORE" >> $GITHUB_ENV
           echo "NEXT_VERSION_ID=$NEXT_VERSION_ID" >> $GITHUB_ENV
+
       - uses: actions/checkout@v4
```

Comment:
Nit: This seems like a case for matrix so the repeated parts of this action can be reused.